am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/Resource.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
```

Comment:
Can we remove that constructor? Builder does not need it, I believe it generates private one under the hood.